OTRExporter/extract_assets.py

```diff
@@ -30,11 +32,12 @@ def main():
         xmlVer = "GC_NMQ_PAL_F"
     elif (args.version == "dbg_mq"):
         xmlVer = "GC_MQ_D"
+        romPath = "..\\soh\\baserom_mq.z64"
```

Comment:
I don't think this code is going to work. You're changing `romPath` after `execStr` has already read from it. It's going to always use `"..\\soh\\baserom_non_mq.z64"`.

Reply:
The execution order is correct.

Main()->buildotr()

Reply:
My mistake, GitHub was presenting the changes in an odd way in the `Files Changed` tab that made it look like those two were part of one function.

Reply:
Ah, that's pretty funny actually.